docs/pages/build-reference/variables.mdx

```diff
@@ -125,6 +125,8 @@ You can create up to 100 account-wide secrets for each Expo account and 100 app-
 
 You can manage secrets through the Expo website and EAS CLI.
 
+> If you are using EAS Build for building in your local infraestructure (`eas build --local`), remember to also add the variables in `~/.bash_profile` (or `~/.zshrc` if you use Zsh)
```

Comment:
```suggestion
> If you are using `eas build --local`, you will not be able to use EAS Secrets. Instead, define your environment variables in your shell profile (for example: `~/.bash_profile` or `~/.zprofile`)
```

Reply:
I think this is not correct. Because... If I use `--local`, I CAN actually use Secrets at the same time.
My situation for example: I have an old Macbook Pro 2015. I build `iOS` on Expo "cloud", and Android on `--local`. So, I actually CAN use it. I just need to define the env vars in both places if you are using both places.

Reply:
sure, you can use secrets with `eas build` without the `--local` flag, but whenever you run it with `--local` you cannot use secrets

Reply:
You can. It's just not gonna work for that local build. But it will work for cloud builds.